### PR TITLE
UCL-550 Added disableAddNew to collumn metadata, updated storybook

### PR DESCRIPTION
**Pull request by lubos-tomsa**

### CODE REVIEW

**Comment by lubos-tomsa** [comment]
Note that Snyk failures are fixed in UCL-332 so it's possible to ignore them in this PR.

File: src/components/DataGrid/DataGridFilters/DataGridToolbar.test.tsx [hunk 1]
```diff
@@ -119,6 +119,56 @@ describe("DataGridToolbar should render", () => {
     expect(secondValueSelect).toHaveTextContent("");
   });
 
+  fdescribe("'Create new' button enabled/disabled feature", () => {
```
Comment (szmejk) [inline src/components/DataGrid/DataGridFilters/DataGridToolbar.test.tsx:122]: should probably be `describe`

File: src/components/DataGrid/DataGridFilters/DataGridToolbar.test.tsx [hunk 2]
```diff
@@ -119,6 +119,56 @@ describe("DataGridToolbar should render", () => {
...
+      await userEvent.click(multiSelectButton);
+    });
+
+    it("should not show 'create new' button if disabled in metadata", async () => {
+      const { getByText, getByLabelText, getByRole } = await setUpFilterForColumn("Type");
+
+      const multiSelectInput = getByRole("combobox");
+      await userEvent.type(multiSelectInput, "Something");
+
+      expect(() => getByText("create new", { exact: false })).toThrow();
```
Comment (szmejk) [inline src/components/DataGrid/DataGridFilters/DataGridToolbar.test.tsx:168]: There's a more standard way of testing [if elements are not present in RTL](https://testing-library.com/docs/guide-disappearance/#asserting-elements-are-not-present). Could you please check it out?

**Review by szmejk — CHANGES_REQUESTED** [review]

File: src/components/DataGrid/DataGridFilters/DataGridToolbar.test.tsx (hunk 1 above)
Comment (lubos-tomsa) [reply src/components/DataGrid/DataGridFilters/DataGridToolbar.test.tsx]: fixed; yes, this was a forgotten focused describe, I was used to use these & to have a linter rule not to forget it :D

File: src/components/DataGrid/DataGridFilters/DataGridToolbar.test.tsx (hunk 2 above)
Comment (lubos-tomsa) [reply src/components/DataGrid/DataGridFilters/DataGridToolbar.test.tsx]: fixed, thanks, I thought there should be a better option

**Review by szmejk — APPROVED** [review]

**Pull request merged**